samples/highcharts/blog/area-bar-iot-devices/demo.js

```diff
@@ -214,7 +214,7 @@ let chartTwo = {
     series: []
 };
 
-fetch('https://cdn.jsdelivr.net/gh/highcharts/highcharts/samples/data/iot-devices.csv').then(function (response) {
+fetch('https://www.highcharts.com/samples/data/iot-devices.csv').then(function (response) {
```

Comment:
AFAIK Goran change it to cover all karma tests, so replacing it with highcharts.com can broke something.

Please double check it with Goran.